lms/templates/staff_problem_info.html

```diff
@@ -128,8 +128,8 @@ <h3>${_('Actions')}</h3>
       <h2>${_("Submission History Viewer")}</h2>
     </header>
     <form id="${element_id}_history_form">
-      <label for="${element_id}_history_student_username">${_("User:")}</label>
-      <input tabindex="0" id="${element_id}_history_student_username" type="text" placeholder=""/>
+      <label for="${element_id}_history_student_username">${_("Learner's edX email address or username:")}</label>
```

Comment:
rather than hardcoding `edX` here -- which could cause issues for opensource, let's use `settings.PLATFORM_NAME`